terraform/main.tf

```
@@ -204,6 +211,24 @@ data "aws_iam_policy_document" "read_datadog_api_key_secret" {
   }
 }
 
+module "grants_prepared_dynamodb_table" {
+  source  = "cloudposse/dynamodb/aws"
+  version = "0.32.0"
+  context = module.this.context
+
+  name                          = "prepareddata"
+  hash_key                      = "grant_id"
+  table_class                   = "STANDARD"
+  enable_streams                = true
+  stream_view_type              = "NEW_AND_OLD_IMAGES"
+  enable_point_in_time_recovery = true
+  enable_encryption             = true
+}
+
+resource "aws_dynamodb_contributor_insights" "grants_prepared_dynamodb_main" {
+  table_name = module.grants_prepared_dynamodb_table.table_name
+}
```

We have a similar case where LocalStack doesn't support `aws_scheduler_schedule` resource types that we're handling in the `DownloadGrantsGovDB` module (although thankfully this workaround will be simpler since nothing else references/depends on this resource). I'd recommend using the same general approach we used there: using an input variable to toggle creation of the resource via `count`:
```suggestion
resource "aws_dynamodb_contributor_insights" "grants_prepared_dynamodb_main" {
  count = var.dynamodb_contributor_insights_enabled ? 1 : 0

  table_name = module.grants_prepared_dynamodb_table.table_name
}
```

In `variables.tf`, the `dynamodb_contributor_insights_enabled` var can have a default value of `true`, and then just set it to `false` in `local.tfvars`. For reference, see the [`var.eventbridge_scheduler_enabled` declaration](https://github.com/usdigitalresponse/grants-ingest/blob/main/terraform/variables.tf#L180-L184) and its [`local.tfvars` override](https://github.com/usdigitalresponse/grants-ingest/blob/main/terraform/local.tfvars#L9).

---

oh cool! That works for me, added this in my latest commit.